api/src/main/java/io/grpc/ClientStreamTracer.java

```diff
@@ -27,6 +27,11 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2861")
 @ThreadSafe
 public abstract class ClientStreamTracer extends StreamTracer {
+  /**
+   * The call was delayed due to waiting for name resolution result.
+   */
+  public static final CallOptions.Key<Boolean> isResolutionDelay =
+      CallOptions.Key.createWithDefault("Name resolution delay", false);
```

Comment:
Name the debug string the name of this API (`io.grpc.ClientStreamTracer.isResolutionDelay`), so it is easier to find/more clear.